src/main/java/org/logstash/plugins/inputs/http/NettyHttpServer.java

```diff
@@ -73,7 +80,9 @@ public void run() {
     public void close() {
         try {
             // stop accepting new connections first
-            processorGroup.shutdownGracefully(0, 10, TimeUnit.SECONDS).sync();
+            bossGroup.shutdownGracefully().sync();
```

Comment:
review note: As my understanding, `ServerBootstrap` internally uses `SingleThreadEventExecutor` and with 0 quiet period, netty may [directly terminate the tasks](https://github.com/netty/netty/blob/4.1/common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java#L772), so let's align with default to get better user experience.